cases/v41/const.py

```diff
@@ -17,22 +16,10 @@
                 18382, 18383, 18384, 18377, 18385, 
                 18386, 18387, 18390]
 
+common_tools = ['check_subscription_rhsm', 'check_subscription_key', 'check_subscription_password']
+common_tools_id = [18412, 18413, 18414]
```

Comment:
for test scenario reason, tools cases are splited into two parts: common_tools are used by test_common_tools.py, while common_tools_subscription are used by test_common_tools_subscription.py. So here should not merge them and remove 18410, 18416 cases.

Reply:
These changes are not from me. Not sure how they appeared here

Reply:
That's much weird. I have to revert this file after merge your commits.